tests/acceptance/addons-test.js

```diff
@@ -399,6 +403,9 @@ module('Acceptance: Addons', function(hooks) {
 
     test('can view a scoped addon with a / in the URL', async function(assert) {
       await visit('/addons/@foo-bar/test-addon');
+
+      await percySnapshot('/addons/show');
```

Comment:
I think this should be in a different test that will have all the addon sections filled out?

Reply:
Right on, will update. Thanks!